app/views/pets/index.html.erb

```diff
@@ -7,6 +7,8 @@
   <h2>Sex: <%= pet.sex %></h2>
   <h2>Shelter: <%= link_to "#{pet.shelter.name}", "/shelters/#{pet.shelter.id}" %></h2>
   <%= link_to "Edit #{pet.name}", "/shelters/#{pet.shelter.id}/pets/#{pet.id}/edit" %> /
-  <%= link_to "Delete #{pet.name}", "/shelters/#{pet.shelter.id}/pets/#{pet.id}", method: :delete %>
+  <% if  pet.adopted? != "Pending" %>
```

Comment:
`delete` button is only available when the `pet` has not been adopted yet (ie. when its application has not been approved yet)